ipxe/dhcp_options.go

```diff
@@ -68,16 +68,18 @@ func Setup(rep *dhcp4.Packet) {
 	rep.SetIP(dhcp4.OptionLogServer, conf.PublicSyslogIPv4) // Have iPXE send syslog to me.
 }
 
-var packetVersion = []byte{1, 0, 255}
+// Version we override in our iPXE build so we can differentiate between vendor/nic builtins vs ours (with the features we need/want)
+// see VERSION_PATCH=255 in ./ipxe/ipxe/build.sh:43
+var ouriPXEVersion = []byte{1, 0, 255}
 
-// IsPacketIPXE returns bool depending on if the request originated with packet's ipxe build
-func IsPacketIPXE(req *dhcp4.Packet) bool {
+// IsOuriPXE returns bool depending on if the request originated from our custom built/embedded iPXE
+func IsOuriPXE(req *dhcp4.Packet) bool {
 	// TODO: make this actually check for iPXE and use ipxe' build system's ability to set name.
-	// This way we could set to something like "Packet iPXE" and then just look for that in the identifier sent in dhcp.
+	// This way we could set to something like "Boots iPXE" and then just look for that in the identifier sent in dhcp.
```

Comment:
Can https://github.com/tinkerbell/boots/blob/4615ecf55f49f1666333a84b6009a0411bf4e2da/job/ipxe.go#L73 be modified to `Boots iPXE` too?

Reply:
yes good catch